tests/API/XCCDF/unittests/test_remediation_kickstart_expected_raw.cfg

```diff
@@ -0,0 +1,56 @@
+# Create partition layout scheme (required for security compliance)
```

Comment:
Why are these headers kept in the raw output?

Reply:
After another round of discussion with Satellite , I've changed the way we format sections in raw mode. It does not have comment headers now, but for partitioning scheme it has pseudo-sections `# %partitions`, `# %logvols` closed with `# %end` to make it easier for them to parse.